app/assets/stylesheets/psique/structures/_menu_lvl2.scss

```diff
@@ -0,0 +1,88 @@
+$menu_lvl2_height: 43px;
+
+.menu-lvl2 {
```

Comment:
Aquí, pero tambien en el resto de estilos en general, las propiedades deben estar ordenadas, mira:

https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder

```
Sort properties in a strict order. By default, will require properties be sorted in alphabetical order, as it's brain dead simple (highlight lines and execute :sort in vim), and it can benefit gzip compression.
```